CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### 7.2.0 (in-progress)
+- Features:
+	- (For test purposes only) Added a way to set subscription ID and config URL in runtime [SDKCF-5614]
```

Comment:
Please update the readme/ userguide.
According to investigation document comment, _"it shouldn't be debug-only functionality - instead it should be provided as a fully-supported runtime override for the build-time config."_

Reply:
Ok. I thought that comment was for Android only.

Reply:
@maureenorea-clores Done. Please check